examples/create-react-app-with-flow/src/pages/index.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React, { Component } from 'react';
+import Button from 'material-ui/Button';
+import Dialog, {
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+} from 'material-ui/Dialog';
+import Typography from 'material-ui/Typography';
+import withStyles from 'material-ui/styles/withStyles';
+import withRoot from '../components/withRoot';
+
+const styles = {
+  root: {
+    textAlign: 'center',
+    paddingTop: 200,
+  },
+};
+
+type DefaultProps = {
+  classes: Object,
+};
+
+export type Props = {};
+
+type State = {
+  open: boolean,
+};
+
+class Index extends Component<DefaultProps & Props, State> {
```

Comment:
Do we need a complex example for this one? Maybe a simple Button is enough.

Reply:
It may be preferable to have something exactly the same as the other without.  That way differences are notable.

Reply:
👍

Reply:
Seriously, a small tutorial explaining the diffs shold be the enough. I did't understood, there is other stuffs (jss, recompose, etc.) disrupting the example. And I couldn't find what was the exactly thing needed to do

Reply:
@joseneto-fixeads to each his own.  BTW - we are dropping flow support as we speak.  If you are starting out, please consider typescript.  More information in #9312